inst/help/BayesianProcess.md

```diff
@@ -0,0 +1,136 @@
+- Parameter Estimates: Determines which parameter estimates are shown in the output. Requesting intercepts adds the mean structure to the model, which adds parameters and will therefore affect some fit indices (such as the information criteria AIC and BIC).
+- Path plots: Displays path plots in the output.
+	- Conceptual: Shows a conceptual path plot. These plots contain no parameter labels or estimates and simplify moderation effects in the model structure by omitting direct effects and interactions. Instead moderators point to the path they are moderating. These plots can be shown even when the model is not complete yet (when `Input type` is `Model number`) and are easier to understand than the statistical path plots.
+	- Statistical: Shows a statistical path plot including parameter labels or estimates for all paths.
+
+#### Options
+
+- Burnin: Number of burnin samples drawn for each MCMC chain.
+- Samples: Number of samples drawn for each MCMC chain.
+- Chains: Number of chains in the MCMC algorithm.
+- Repeatability: Seed to exactly reproduce the output.
+- Prior distributions: Show the prior distributions for path coefficients in the output. The distributions can be modified in the `Advanced` section.
+- Parameter labels: Display the labels of parameter in the output tables. For indirect and direct (total) effects, display the equations for the effects.
+- Lavaan syntax: Show the lavaan syntax used to fit the model in the output.
+- Credible intervals: The level of credible intervals in the output tables. Credible intervals are based on quantiles of the posterior samples.
+
+#### Plots
+
+- Color palette: Color palette for plots.
+- MCMC plots: Options for displaying posterior distributions and MCMC diagnostics.
+	- Aggregate chains for densities and histograms: If checked, the samples of different chains are aggregated in density plots and histograms. If unchecked, there are separate colors per chain.
+	- Density: Show a density plot of the posterior samples.
+	- Histogram: Show a histogram of the posterior samples.
+	- Trace: Show a trace plot of the posterior samples.
+	- Autocorrelation: Plot the autocorrelation of the posterior samples.
+		- No. lags: the maximum number of lags to show in the autocorrelation plot.
+		- Type: whether to display the autocorrelation as a bar at each lag, or as a line that connects subsequent lags.
+	- Bivariate scatter: Show a bivariate scatter plot of all pairs of variables. Only shows output when more than 1 parameter is sampled.
+		- Diagonal plot type: Show a density plot or a histogram on the diagonal entries of the scatter plot.
+		- Off-diagonal plot type: Show a hexagonal bivariate density plot, or a contour plot on the off-diagonal entries of the scatter plot.
+- Path plots: Options for path plots.
```

Comment:
move this part above the MCMC explanation, following the structure in the plot tab